pkg/operator/controller/status/controller.go

```diff
@@ -268,6 +277,15 @@ func (r *reconciler) getOperatorState(ingressNamespace, canaryNamespace string)
 		state.IngressControllers = ingressList.Items
 	}
 
+	if len(co.Status.Extension.Raw) > 0 {
+		extension := &IngressOperatorStatusExtension{}
+		if err := json.Unmarshal(co.Status.Extension.Raw, extension); err != nil {
+			return state, fmt.Errorf("failed to unmarshal status extension of cluster operator %q: %w", co.Name, err)
+		} else {
+			state.UnmanagedGatewayAPICRDNames = extension.UnmanagedGatewayAPICRDNames
```

Comment:
`extension` is declared outside the `if` statement's scope, so you don't need the `else`:
```suggestion
		}
		state.UnmanagedGatewayAPICRDNames = extension.UnmanagedGatewayAPICRDNames
```